src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -4,28 +4,35 @@
 
 #define FLAGS (ACTOR_FLAG_0 | ACTOR_FLAG_2 | ACTOR_FLAG_4)
 
+typedef enum {
+    ENSKB_ACTION_BURIED,
+    ENSKB_ACTION_DYING,
+    ENSKB_ACTION_DAMAGED,
+    ENSKB_ACTION_ATTACKING,
+    ENSKB_ACTION_WALKING,
+    ENSKB_ACTION_RECOILING,
+    ENSKB_ACTION_STUNNED
+} EnSkbActionState;
```

Comment:
the term "action state" has always felt pretty redundant to me. To me, an action is a very specific kind of state, so saying both is not necessary

In the armos I used the term "behavior" to describe this kind of variable. That is, it describes the general behavior of the actor without being tied to a specific action/action function.
Behavior might not be the best term for this, but its a pattern that is somewhat common among enemies and itd be good to make it consistent

Reply:
that's fair, it looks like Armos is currently the only one following this but I am all for alignment on it going forward, I'll take consistency over 100%-correct any day. changed

Reply:
yea behavior is def not common. I dont think we have a decided on term for this just yet. was moreso just putting my thoughts out there on it. we will prob have mroe discussion and be consistent about it soon